app/app.go

```diff
@@ -1308,6 +1308,10 @@ func (app *App) ProcessBlock(ctx sdk.Context, txs [][]byte, req BlockProcessRequ
 	prioritizedResults, ctx := app.BuildDependenciesAndRunTxs(ctx, prioritizedTxs)
 	txResults = append(txResults, prioritizedResults...)
 
+	// Finalize all Bank Module Transfers here so that events are included for prioritiezd txs
```

Comment:
note that we need this because SOME prioritized transactions can pay fees (oracle doesn't but some of the dex ones do) so we need to make sure to flush those BEFORE we create the new context memcache for the rest of the transactions cc @BrandonWeng

Reply:
If possible can we add a test to enforce this?